lib/client/rpc/dao.ts

```diff
@@ -118,47 +118,72 @@ export class FutarchyRPCDaoClient implements FutarchyDaoClient {
         ...daoAccount,
         tokenMint: daoAccount.tokenMint
           ? daoAccount.tokenMint
-          : new PublicKey(daoAccount.metaMint ?? ""),
+          : new PublicKey(daoAccount.metaMint ?? "")
       };
       return {
         daoAccount: dao,
         baseToken,
         quoteToken,
-        protocol,
+        protocol
       };
     }
   }
-  async getMinLpToProvide(daoAggregate: DaoAggregate): Promise<{ base: BN, quote: BN } | undefined> {
-    const autocrat = new Program<AutocratV1>(AUTOCRAT_V1_IDL, "5scUH1qY87Awh5D8bQkbP2uv9BaXJXiMgrqRBw1K2vVv", this.rpcProvider)
-    const currentDao = daoAggregate.daos.filter((dao) => dao.protocol.deploymentVersion === "V1").slice(-1)[0]
-    if (!currentDao) return
+  async getMinLpToProvide(
+    daoAggregate: DaoAggregate
+  ): Promise<{ base: BN; quote: BN } | undefined> {
+    const autocrat = new Program<Autocrat>(
+      AUTOCRAT_V0_3_IDL,
+      "5scUH1qY87Awh5D8bQkbP2uv9BaXJXiMgrqRBw1K2vVv",
```

Comment:
`AMM5G2nxuKUwCLRYTW7qqEwuoqCtNSjtbipwEmm2g8bH` is the new program

Reply:
added new autocrat